nd2reader/raw_metadata.py

```diff
@@ -160,7 +161,16 @@ def _parse_z_levels(self):
         Returns:
             list: the z levels, just a sequence from 0 to n.
         """
-        return self._parse_dimension(r""".*?Z\((\d+)\).*?""")
+        z_levels = self._parse_dimension(r""".*?Z\((\d+)\).*?""")
+        if 0 == len(z_levels):
+            z_levels = parse_if_not_none(self.z_data, self._parse_z_coordinates)
+            if isinstance(z_levels, type(None)):
```

Comment:
Could you replace this by `if z_levels is None:`? Thanks!

Reply:
Sure, I have amended this.